solution/ui/regulations/eregs-vite/src/views/Resources.vue

```diff
@@ -21,11 +21,21 @@
                         @click:append="executeSearch"
                         @click:clear="clearSearchQuery"
                     />
-                    <div class="search-suggestion" v-if="multiWordQuery">
-                        Didn't find what you were looking for? Try searching for
-                      <a @click="doQuoteSearch">"{{this.searchQuery}}"</a>
+                    <div class="search-suggestion">
+                        <div v-if="multiWordQuery">
+                            Didn't find what you were looking for? Try searching for
+                            <a @click="doQuoteSearch">"{{this.searchQuery}}"</a>
+                        </div>
+                        <div class="synonyms" v-if="synonyms.length > 0"> 
+                            <span v-if="multiWordQuery">Or </span>Search for similar terms:
+                            <span v-bind:key=a v-for="a in synonyms">
+                                "<a @click="synonymLinks(a)">{{a}}</a>"
```

Comment:
I think no quotes around the synonym links per Britta's comment in slack.

Reply:
Removed